src/main/java/org/opentripplanner/routing/edgetype/WheelchairEdge.java

```diff
@@ -0,0 +1,5 @@
+package org.opentripplanner.routing.edgetype;
+
+public interface WheelchairEdge {
```

Comment:
This need a better name and JavaDoc in interface and methods. This is not an Edge, witch the name suggest.

Reply:
I've renamed it to `WheelchairTraversalInformation` but I'm not 100% happy with it. Do you have a better idea?